test/integration-test.py

```diff
@@ -241,6 +245,52 @@ def run_client_tests():
     cmd = os.path.join(root, 'tests', 'boulder-integration.sh')
     run_custom(cmd, cwd=root)
 
+
+def get_future_output(cmd, date, cwd=None):
+    return subprocess.check_output('FAKECLOCK=`date -d "%s"` %s' % (date.isoformat(), cmd), cwd=cwd, shell=True)
+
+def run_expired_authz_purger_test(challenge_types):
+    # Issue the certificate and transform it from DER-encoded to PEM-encoded.
+    exit_code = subprocess.Popen('''
```

Comment:
Since we're returning success from test.js with abort-step now, we can use check_output here and simplify things a bit.